src/Stratis.Bitcoin/Configuration/DataFolder.cs

```diff
@@ -27,6 +27,7 @@ public DataFolder(string path)
             this.RpcCookieFile = Path.Combine(path, ".cookie");
             this.WalletPath = Path.Combine(path);
             this.LogPath = Path.Combine(path, "Logs");
+            this.ApplicationsPath = Path.Combine(path, "apps");
```

Comment:
Ah ok ignore my last comment on the path

Reply:
Yep